### PR TITLE
docs: november release notes

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,101 +1,80 @@
...
+ADD COLUMN IF NOT EXISTS dnssuffix varchar(256);
+```
+
 :material-new-box:** Feature: AMT Maintenance commands **
 
-Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
+We have added a maintenance task that will enable syncing the OS Hostname and AMT Hostname.  When this command is run, if the OS and AMT hostnames are different, the OS hostname will be applied to AMT.  This enables AMT to respond to network requests that use the fully qualified domain name (fqdn) of the device.
 
-:material-star:** Customer Request: Unprovision AMT devices via MPS **
+:material-star:** Customer Request: Health check improvement **
```
Comment (rsdmike) [inline docs/release-notes.md:33]: Service Startup Improvement

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,101 +1,80 @@
...
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS friendlyname varchar(256);
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS dnssuffix varchar(256);
+```
+
 :material-new-box:** Feature: AMT Maintenance commands **
 
-Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
+We have added a maintenance task that will enable syncing the OS Hostname and AMT Hostname.  When this command is run, if the OS and AMT hostnames are different, the OS hostname will be applied to AMT.  This enables AMT to respond to network requests that use the fully qualified domain name (fqdn) of the device.
```
Comment (rsdmike) [inline docs/release-notes.md:31]: fqdn => FQDN

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,101 +1,81 @@
...
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS friendlyname varchar(256);
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS dnssuffix varchar(256);
+```
+
 :material-new-box:** Feature: AMT Maintenance commands **
 
-Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
+We have added a maintenance task that will enable syncing the OS Hostname and DNS Suffix with AMT Hostname and DNS Suffix.  When this command is run, if the OS and AMT host names are different, the OS hostname will be applied to AMT.  This enables AMT to respond to network requests that use the fully qualified domain name (FQDN) of the device.
```
Comment (Kimberly-Davis) [inline docs/release-notes.md:31]: We added a maintenance task command that enables syncing the OS hostname and DNS Suffix with AMT hostname and DNS suffix.  If the OS and AMT hostnames are different when the command is run, the OS hostname will be applied to AMT.  This maintenance task enables AMT to respond to network requests that use the fully qualified domain name (FQDN) of the device.

Hey @matt-primrose, do you want to mention the name of the command here?

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,101 +1,81 @@
...
 
-:material-star:** Customer Request: Unprovision AMT devices via MPS **
+:material-star:** Customer Request: Service startup improvement **
 
-We've had several customer requests to add this feature to MPS and this release we are delivering.  We've added an API endpoint ( DELETE /api/v1/amt/deactivate/{guid} ) to MPS that will send the unprovision command to a connected AMT device and will then remove that AMT device from the MPS database.  In order to reconnect that AMT device to MPS again, RPC and RPS will need to be used.
+Both RPS and MPS will now check for dependent services to be available before starting up.  If a dependent service is not available, they will wait and retry after a few seconds.  If after 10 attempts the service will shutdown.  In a production deployment, this shutdown will typically trigger a restart of the service starting the cycle again until dependent services are available. Retry attempts will be listed in the service logs.
```
Comment (Kimberly-Davis) [inline docs/release-notes.md:35]: Both RPS and MPS now check for dependent services available before starting up.  If a dependent service is not available, the RPS and MPS wait and retry after a few seconds.  After 10 retry attempts the service shuts down.  In a production deployment, this shutdown typically triggers a restart of the service, starting the cycle again until dependent services are available. Retry attempts are listed in the service logs.

File: docs/release-notes.md [hunk 5]
```diff
@@ -1,101 +1,81 @@
...
 
-:material-star:** Customer Request: Enable local Wi-Fi network synchronization by default **
+:material-star:** Customer Contributions: Several MPS issues filed and fixed! **
 
-We have updated RPS to enable AMT to sync Wi-Fi networks automatically if a local management service (LMS) is running.  This means if a user connects to a wireless network in the OS, this wireless network will automatically be synced to AMT.  When the OS is down, AMT will use this wireless network to connect to the network so that the device can be remotely managed.
+A big thank you to Orin from [Laplink](https://www.laplink.com/) for both filing and fixing several MPS issues!  We love getting customer feedback as well as customer contributions.  When we get customers filing issues and then immediately contributing the fix, well, that is what open source is all about!
```
Comment (Kimberly-Davis) [inline docs/release-notes.md:39]: A big thank you to Orin from [Laplink](https://www.laplink.com/) for both filing and fixing several MPS issues!  We love getting customer feedback as well as customer contributions.  Customers filing issues and then immediately contributing the fix, well, that is what open source is all about!

**Review by Kimberly-Davis — COMMENTED** [review]
Hey,

Overall great.  A very light edit, Matt. Where I could, I tried to reduce a few words. You've all decided to make this a very personal, authentic-sounding release notes doc, and I want to support you in that. Casual can get slightly verbose though, and it's what make so much online writing and instructional YouTube videos insufferable. 

Anyway, looks like the team is doing great. 

Regards,
Kimberly

**Review by rsdmike — APPROVED** [review]

**Pull request merged**